extensions/amp-story-interactive/0.1/amp-story-interactive-binary-poll.css

```diff
@@ -156,7 +157,7 @@
 
 .i-amphtml-story-interactive-post-selection
   .i-amphtml-story-interactive-option-percentage-text {
-  transform: scale(1) !important;
+  transform: scale(1) translateY(-0.75em) !important;
```

Comment:
I'm not sure if this is always needed:
<img width="445" alt="Screen Shot 2020-10-14 at 3 31 45 PM" src="https://user-images.githubusercontent.com/3860311/96036467-822be300-0e32-11eb-921c-c1164702a1f3.png">